backend/config.env

```diff
@@ -0,0 +1,2 @@
+ATLAS_URI=mongodb+srv://accenturetest:accenturepwd@knowledge-accenture.ffhbf.mongodb.net/?retryWrites=true&w=majority
```

Comment:
Although I don’t know if we want to keep this in the repo for everyone to see 👁️